SingularityService/src/main/java/com/hubspot/singularity/resources/TaskResource.java

```diff
@@ -609,4 +613,30 @@ private SingularityTaskShellCommandRequest startShellCommand(SingularityTaskId t
     SingularityTaskId taskIdObj = getTaskIdFromStr(taskId);
     return taskManager.getTaskShellCommandUpdates(new SingularityTaskShellCommandRequestId(taskIdObj, commandName, commandTimestamp));
   }
+
+  @GET
+  @Path("/download/{httpPrefix}/{slaveHostname}/port/{port}/path/{path}")
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  @Operation(summary = "Proxy a file download from a Mesos Slave through Singularity")
+  @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
```

Comment:
Since this is a `GET`, it shouldn't consume anything.

Reply:
-- whoops -- left this there from when I had it as a `POST`